### PR TITLE
Link docs blog article

### PR DESCRIPTION
**Pull request by maximiliankolb**

### CODE REVIEW

**Comment by maximiliankolb** [comment]
This PR is now rfr. I've added links to the orcharhino documentation and ACD ELK blog article. If you're okay with my other changes (mostly formatting and grammar), I would rebase before you can merge it.

@Fobhep

File: README.md [hunk 1]
```diff
@@ -52,20 +51,18 @@ firewall_ports:
...
 
-> `es_add_repository` Setting this to `false` will stop Ansible from adding the official Elastic package repositories (if 
-> es_use_repository is true) if you want to use a repo already present"
+As stated in [github.com/elastic/ansible-elasticsearch](https://github.com/elastic/ansible-elasticsearch):
+> "`es_use_repository`: Setting this to `false` will stop Ansible from using the official Elastic package from any repository configured on the system.
 
+> `es_add_repository`: Setting this to `false` will stop Ansible from adding the official Elastic package repositories (if `es_use_repository` is `true`) if you want to use a repository already present".
 
-`es_add_repository` set via `group_vars/all.yml` will also influence the existence of the Elasticsearch/Kibana repository on Kibana hosts. The default value inside the `kibana` role is `true`.
+> `es_add_repository`: Set via `group_vars/all/main.yml` will also influence the existence of the Elasticsearch/Kibana repository on Kibana hosts. The default value inside the `kibana` role is `true`.
```
Comment (Fobhep) [inline README.md]: This is not a direct quote from another repo as the two points above are, so I would not use Markdown Quote syntax here

**Review by Fobhep — CHANGES_REQUESTED** [review]
lgtm - except the one quote thing

File: README.md (hunk 1 above)
Comment (maximiliankolb) [reply README.md]: Oh, my bad. Will fix and rebase.

**Comment by maximiliankolb** [comment]
Rebased.

**Comment by Fobhep** [comment]
lgtm

**Review by Fobhep — APPROVED** [review]

**Pull request merged**